src/libs/actions/Session/index.ts

```diff
@@ -184,6 +184,13 @@ function hasStashedSession(): boolean {
     return Boolean(stashedSession.authToken && stashedCredentials.autoGeneratedLogin && stashedCredentials.autoGeneratedLogin !== '');
 }
 
+/**
+ * Checks if the user is authenticated
+ */
+function isAuthenticated(): boolean {
```

Comment:
Can we change this to `hasAuthToken`? I just don't want this method to be used improperly in the future to have people think that the user has actual permissions since this is really just "do they have an auth token" not "are they actually logged in"